esp/services/ws_machine/ws_machineService.cpp

```diff
@@ -1183,7 +1183,7 @@ int Cws_machineEx::runCommand(IEspContext& context, const char* sAddress, const
         IFRunSSH * connection = createFRunSSH();
         connection->init(command.str(),NULL,NULL,NULL,m_SSHConnectTimeoutSeconds,0);
         // executed as single connection
-        connection->exec(sAddress,NULL,true);
+        connection->exec(sAddress, NULL, false);
```

Comment:
sAddress here is a const char *, but it appears the definition is:
```
    void exec(
              const IpAddress &ip,
              const char *workdirname,
              bool _background)
```
Where the first arg is a const IpAddress &

Reply:
Talked with @jakesmith about this - we should add an 'explicit' to the IpAddress() ctor:
```
    explicit IpAddress(const char *text)                         { ipset(text); }
```
In jsocket.hpp to make this more correct.  And then we can no longer pass in sAddress here, but need to first create an IpAddress type and init with the sAddress text and pass that in.

Reply:
Thanks @mckellyln 

I think this PR is good to merge now.